filebeat/input/v2/compat/compat_test.go

```diff
@@ -110,7 +110,6 @@ paths:
 prospector:
   scanner:
     symlinks: true
-take_over: true
```

Comment:
This is not used at all in this part of the code, so I just removed the legacy config format.